core-java-modules/core-java-21/src/test/java/com/baeldung/java21/EmojiSupportUnitTest.java

```diff
@@ -0,0 +1,69 @@
+package com.baeldung.java21;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.junit.jupiter.api.Test;
+
+class EmojiSupportUnitTest {
+
+    @Test
+    public void testIsEmoji() {
```

Comment:
Test names must be BDD style, either:
- given_when_then
- when_then

and in JUnit5, you can remove the public modifier

Reply:
review incorporated.